Reference/Searching/Examine/Quick-Start/index-v8.md

```diff
@@ -0,0 +1,260 @@
+---
+versionFrom: 8.0.0
+versionTo: 8.0.0
+---
+
+# Quick start
+
+_This guide will help you get setup quickly using Examine with minimal configuration options. Umbraco ships Examine with 3 indexes: internal, external, and members. The internal index should not be used for searching when returning results on a public website because it includes content that has not been published yet. Instead you can use the external index to get up and running._
+
+:::note
+This document has been verified for Umbraco 8.
+
+If you are using Umbraco 9 or later versions, please refer to the note on the [Examine documentation landing page](index.md) for more details.
+:::
+
+## Performing a search
+
+:::note
+In the coming examples the Umbraco Starter Kit has been used, as it provides some example content that can be searched. Some of the examples below therefore may require 'the setting up of templates, etc' if you are following the guide on your own existing site.
+:::
+
+The starter kit comes with some Templates, Document Types and content nodes created already. We will use some of these to set up a basic search system. This is a 'Quick Start' guide, as much more complex searches are possible with Examine.
+
+We will make it possible to 'search' on the _People_ page, by adding a search bar to the template page: `people.cshtml` - add the following form at the top of the template, but underneath the `<nav>` element:
+
+```csharp
+...
+</nav>
+-->
+<div>
+    <form action="@Model.Url()" method="get">
+        <input type="text" placeholder="Search" id="query" name="query" />
+        <button>Search</button>
+    </form>
+</div>
+<div class="employee-grid">
+...
+```
+This will create a basic input field at the top of the page and make it post to the same people page when submitted along with the search term.
+### Retrieving the Search Term
+
+Right below the form, add the following:
+```csharp
+<div>
+    @{
+        var searchTerm = string.Empty;
+        searchTerm = string.IsNullOrEmpty(Request["query"])
+            ? string.Empty
+            : Request["query"];
+        if (searchTerm == string.Empty)
+        {
+            <p>Enter search term</p>
+        }
+        else
+        {
+            //perform the search
+        }
+    }
+</div>
+```
+Here we are getting the request query from the form. If the string is empty we ask them to submit a term, otherwise we will perform the search.
```

Comment:
```suggestion
Here we are getting the request query from the form. If the string is empty, we ask them to submit a term, otherwise, we will perform the search.
```